mandatory/2-function-creation.js

```diff
@@ -25,7 +34,13 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  let noWhiteSpacesString = arrayOfStrings.trim();
+  let noForwardSlashesString = noWhiteSpacesString.replace("/", ""));
+  let lowerCaseString = noForwardSlashesString.toLoweCase();
+
+  return lowerCaseString;
+}
 
```

Comment:
plz. double check this function, and think about the type of the arrayOfStrings, and whether it has trim function